MPower/app/src/main/java/org/sagebase/mpower/SettingsFragment.java

```diff
@@ -1,33 +1,42 @@
 package org.sagebase.mpower;
 
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
 import android.os.Bundle;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.preference.Preference;
 import android.support.v7.preference.PreferenceCategory;
 import android.support.v7.preference.PreferenceScreen;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.EditText;
+
+import org.researchstack.backbone.utils.LogExt;
+import org.sagebionetworks.bridge.researchstack.MPowerPrefs;
 
 public class SettingsFragment extends org.researchstack.skin.ui.fragment.SettingsFragment {
-    public static final String KEY_EXAMPLE = "Sample.EXAMPLE";
+    public static final String KEY_CUSTOM_QUESTION = "MPOWER_CUSTOM_QUESTION";
 
     @Override
     public void onCreatePreferences(Bundle bundle, String s) {
         super.onCreatePreferences(bundle, s);
 
-        // Get our screen which is created in Skin SettingsFragment
         PreferenceScreen screen = getPreferenceScreen();
 
         // Get profile preference
         PreferenceCategory category = (PreferenceCategory) screen.findPreference(KEY_PROFILE);
 
-        // If category exists, we should add mole mapper specific things. If not, that means we
-        // are not consented so we have no data to set.
+        // TODO:  not sure where this should go so adding to profile for now
+        //        also, should this only show after they have entered a question from the daily task?
```

Comment:
That's a good question.  I'd say it should only show once you have set up a custom question.